tests/openqa/webui/test_results.pm

```diff
@@ -57,6 +57,9 @@ sub run {
     for (1 .. 25) {
         send_key 'f5';
         wait_still_screen;
+        send_key 'home';
```

Comment:
What does this `home` key?

Reply:
Replying to myself: It is to go up on the page. Maybe worth to add a commentary?